From b1a5db9318ca33a03f2700758d5debad907d21b6 Mon Sep 17 00:00:00 2001
From: Joel Winarske <joel.winarsk@gmail.com>
Date: Wed, 19 Jul 2023 15:26:40 -0700
Subject: [PATCH] remove explicit x11 dependency

This is superfluous and may conflict with the actual rendering system used in
gtk3 (e.g. wayland.)

Upstream: https://github.com/flutter/flutter/issues/123073
Signed-off-by: Joel Winarske <joel.winarske@gmail.com>
Signed-off-by: Adam Duskett <adam.duskett@amarulasolutions.com>
---
 flutter/shell/platform/linux/config/BUILD.gn | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/flutter/shell/platform/linux/config/BUILD.gn b/flutter/shell/platform/linux/config/BUILD.gn
index da0ac056b..70c7a0ea1 100644
--- a/flutter/shell/platform/linux/config/BUILD.gn
+++ b/flutter/shell/platform/linux/config/BUILD.gn
@@ -5,10 +5,6 @@
 import("//build/config/linux/pkg_config.gni")
 import("//flutter/shell/platform/glfw/config.gni")
 
-pkg_config("x11") {
-  packages = [ "x11" ]
-}
-
 pkg_config("gtk") {
   packages = [ "gtk+-3.0" ]
 }
-- 
2.41.0

